src/Compatibility/Microsoft.DotNet.ApiCompatibility/Abstractions/MetadataInformation.cs

```diff
@@ -25,6 +25,9 @@ public MetadataInformation(string name, string targetFramework, string assemblyI
             DisplayString = displayString ?? assemblyId;
         }
 
+        public override bool Equals(object obj) =>
+            Equals(obj is MetadataInformation) && Equals((MetadataInformation)obj);
```

Comment:
I thought the `is` operator returns a boolean?
This calls `Equals(boolean)`?
Should it be
```C#
obj is MetadataInformation && Equals((MetadataInformation)obj)
```
or perhaps
```C#
obj is MetadataInformation md && Equals(md)
```

Reply:
Oh right, thanks.